src/commands/commander.h

```diff
@@ -185,6 +187,10 @@ inline uint64_t ParseCommandFlags(const std::string &description, const std::str
       flags |= kCmdCluster;
     else if (flag == "no-dbsize-check")
       flags |= kCmdNoDBSizeCheck;
+    else if (flag == "deny-oom")
+      flags |= kCmdDenyOom;
```

Comment:
I think we don't plan to implement it in this PR, so please remove it.

Reply:
I agree with this, thanks for the heads up, I'll remove the flags later on.